test/test_api.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+import os
+
+from behave_testrail_reporter.api import APIClient
+
+
+class APIClientTestCase(unittest.TestCase):
+    def setUp(self):
+        if os.environ.get(u'TESTRAIL_USER'):
+            del os.environ[u'TESTRAIL_USER']
+        if os.environ.get(u'TESTRAIL_KEY'):
+            del os.environ[u'TESTRAIL_KEY']
```

Comment:
The effect of this change will leak out of the scope of this test. I don't think that's a good idea.

Reply:
Perhaps [`patch.dict`](https://docs.python.org/3/library/unittest.mock.html#patch-dict) would help.